core/src/main/java/jenkins/management/AdministrativeMonitorsApi.java

```diff
@@ -12,28 +11,19 @@
 import org.kohsuke.stapler.verb.GET;
 
 import javax.servlet.ServletException;
-import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
-import java.util.Collection;
-import java.util.Collections;
 
 @Extension
 @Restricted(NoExternalUse.class)
 public class AdministrativeMonitorsApi implements RootAction {
     @GET
     public void doNonSecurityPopupContent(StaplerRequest req, StaplerResponse resp) throws IOException, ServletException {
-        resp.setStatus(HttpServletResponse.SC_OK);
-        resp.setContentType("text/html");
```

Comment:
why aren't this needed?

Reply:
It's done automatically by the JellyFacet (need to find exactly the code, inside Stapler, digging...)